docs/content/error/$compile/noctrl.ngdoc

```diff
@@ -0,0 +1,68 @@
+@ngdoc error
+@name $compile:noctrl
+@fullName Controller is required.
+@description
+
+When using the `bindToController` feature of AngularJS, a directive is required
+to have a Controller, in addition to a controller identifier.
```

Comment:
> When using the `bindToController` feature of AngularJS, a directive is required
to provide the `controller as` identifier, in addition to providing a controller.